### PR TITLE
Orchestrator Integration Removal and `orc_client_user` removal

### PR DESCRIPTION
**Pull request by GuptaManan100**
## Description

This PR removes the deprecated orchestrator integration and cleans up the `orc_client_user` user from the `init_db.sql` file since it is no longer required.

## Related Issue(s)

- #6612 

## Checklist

-   [x] "Backport me!" label has been added if this change should be backported
-   [x] Tests were added or are not required
-   [x] Documentation was added or is not required

### CODE REVIEW

File: go/vt/vttablet/tabletmanager/restore.go [hunk 1]
```diff
@@ -30,17 +30,16 @@ import (
 	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/logutil"
 	"vitess.io/vitess/go/vt/mysqlctl"
+	binlogdatapb "vitess.io/vitess/go/vt/proto/binlogdata"
```
Comment (rsajwani) [inline go/vt/vttablet/tabletmanager/restore.go:33]: line 33, 34,35 should be separate section after line 42

File: go/vt/vttablet/tabletmanager/rpc_backup.go [hunk 2]
```diff
@@ -17,18 +17,16 @@ limitations under the License.
...
-	"context"
-
 	"vitess.io/vitess/go/vt/logutil"
 	"vitess.io/vitess/go/vt/mysqlctl"
-	"vitess.io/vitess/go/vt/topo/topoproto"
-	"vitess.io/vitess/go/vt/vterrors"
-
 	tabletmanagerdatapb "vitess.io/vitess/go/vt/proto/tabletmanagerdata"
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	"vitess.io/vitess/go/vt/topo/topoproto"
```
Comment (rsajwani) [inline go/vt/vttablet/tabletmanager/rpc_backup.go:28]: move it with line 25

File: go/vt/vttablet/tabletmanager/rpc_replication.go [hunk 3]
```diff
@@ -29,13 +29,12 @@ import (
...
 	"vitess.io/vitess/go/vt/logutil"
 	"vitess.io/vitess/go/vt/mysqlctl"
-	"vitess.io/vitess/go/vt/servenv"
-	"vitess.io/vitess/go/vt/topo/topoproto"
-	"vitess.io/vitess/go/vt/vterrors"
-
 	replicationdatapb "vitess.io/vitess/go/vt/proto/replicationdata"
 	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
 	"vitess.io/vitess/go/vt/proto/vtrpc"
+	"vitess.io/vitess/go/vt/servenv"
```
Comment (rsajwani) [inline go/vt/vttablet/tabletmanager/rpc_replication.go:35]: same here

**Comment by rsajwani** [comment]
LGTM

**Comment by rsajwani** [comment]
overall , i see imports are bit misplaced but everything else looks good. please fix the imports before merging ..

**Review by rsajwani — APPROVED** [review]

**Comment by GuptaManan100** [comment]
@deepthi I'll merge this PR and remove the replication-manager entirely in a follow-up PR.

**Comment by GuptaManan100** [comment]
`vttablet_prscomplex` test is failing, but the failure is unrelated to this PR. Merging it.

**Pull request merged**